test/test-app.js

```diff
@@ -20,13 +20,6 @@ fastBoot.start({
 
 var orig = {};
 
-orig.statSync = fs.statSync;
-var statSyncCount = 0;
```

Comment:
Debugging showed that `statSyncCount` always remains at `0` now, possibly due to recent optimizations in npm?